packages/salesforcedx-utils-vscode/src/cli/commandExecutor.ts

```diff
@@ -67,6 +67,7 @@ export class CliCommandExecutor {
           GlobalCliEnvironment.environmentVariables
         )
       : options;
+    this.options.env.SFDX_TOOL = 'salesforce-vscode-extensions';
```

Comment:
I think this should be enough. What happens when a user is not in the latest version of the CLI that consumes this change ?

Reply:
Nothing. The CLI does nothing with this envar currently, so it is just a new op. Once they update to the CLI that has the analytic changes, probably next week, we will start tot get the data.